pages/Headings.md

```diff
@@ -0,0 +1,11 @@
+type:: [[Feature]]
+description:: Create document headings
+
+- ## Usage
+- There are multiple ways to create a heading. You can use markdown syntax while editing a block, by adding `#` signs at the beginning of the block, followed by a space character. The heading level will be based on the number of signs that you used. For example, `### Title` will be translated into an `<h3>`.
```

Comment:
Usually we put content under headers so we can reference that content with a block link and collapse it as necessary. I can fix

Reply:
I see. Thanks for pointing that out.